runtime/tr.source/trj9/optimizer/FearPointAnalysis.cpp

```diff
@@ -39,18 +39,26 @@ bool TR_FearPointAnalysis::virtualGuardsKillFear()
    return feGetEnv("TR_FPAnalaysisGuardsDoNotKillFear") == NULL;
    }
 
-static bool containsPrepareForOSR(TR::Block *block)
+bool TR_FearPointAnalysis::containsPrepareForOSR(TR::Block *block)
```

Comment:
Since this function is called from two analyses would it be better somewhere in the OSR infrastructure?